controller/src/beerocks/master/tasks/bml_task.h

```diff
@@ -83,6 +83,9 @@ class bml_task : public task {
         ACS_START_EVENT_AVAILABLE,
```

Comment:
I think this can be squashed to the previous commit, not much benefit from standing on its own.

Reply:
done